### ISSUE TITLE
installation command is incorrect

### ISSUE THREAD

**Issue by obuckley-kenzan** [kenzanlabs/pipeline-validate-css#5]
it should be
```
npm install git+ssh://git@github.com:kenzanmedia/pipeline-validate-css.git
```

it's missing the `://` before `git@`

also, using the `--save-dev` flag saves in package.json as 
```
"pipeline-validate-css": "file:../../../../../../../var/folders/hj/4pr577wx1zvdxxhhmz6386300000gp/T/npm-10495-a7f81e5d/1446218036262-0.05559100350365043/1f01fa2ea07c76c4d5ab29b4bbcc6a8d417eac12",
```

**Comment by thisgeek** [comment]
Odd. What was your .npmrc at the time? (Be sure to omit auth data from it if you post.)

**Comment by obuckley-kenzan** [comment]
Hey @thisgeek 
I had nothing set per se.  If I check in my home directory, there is no file
```
$ ls -la ~/
total 760
drwxr-xr-x+  60 obuckley  staff    2040 Nov  2 20:44 .
drwxr-xr-x    6 root      admin     204 Dec 25  2014 ..
-r--------    1 obuckley  staff       7 Dec 25  2014 .CFUserTextEncoding
-rw-r--r--@   1 obuckley  staff   34820 Nov  2 20:44 .DS_Store
drwx------    2 obuckley  staff      68 Nov  3 08:35 .Trash
drwxr-xr-x    3 obuckley  staff     102 Aug 22 01:14 .WebIde90
drwxr-xr-x    3 obuckley  staff     102 Apr 15  2015 .WebStorm10
drwxr-xr-x    3 obuckley  staff     102 Nov  2 20:44 .WebStorm11
drwxr-xr-x    3 obuckley  staff     102 Jan 12  2015 .WebStorm9
-rw-r--r--    1 obuckley  staff     702 Oct 12 08:45 .anyconnect
-rw-r--r--    1 obuckley  staff  264667 Apr 18  2015 .babel.json
-rw-------    1 obuckley  staff    9619 Nov  2 19:23 .bash_history
-rw-r--r--    1 obuckley  staff     125 May 27 22:30 .bash_profile
-rw-r--r--    1 obuckley  staff      18 Apr 20  2015 .bashrc
drwxr-xr-x    3 obuckley  staff     102 Dec 25  2014 .cache
drwxr-xr-x    4 obuckley  staff     136 May 15 17:19 .cisco
drwx------    5 obuckley  staff     170 May 27 15:09 .config
drwxr-xr-x    3 obuckley  staff     102 Sep 28 13:44 .cordova
drwx------    3 obuckley  staff     102 Jan  8  2015 .cups
drwx------   10 obuckley  staff     340 Nov  3 08:34 .dropbox
drwxr-xr-x    4 obuckley  staff     136 May 20 13:03 .gem
-rw-r--r--    1 obuckley  staff     510 Oct 22 12:24 .gitconfig
-rw-r--r--@   1 obuckley  admin      13 Dec 19  2014 .gitignore_global
drwxr-xr-x    6 obuckley  staff     204 May 27 22:22 .gradle
-rw-r--r--@   1 obuckley  admin      27 Dec 19  2014 .hgignore_global
drwxr-xr-x    3 obuckley  staff     102 Sep 28 13:44 .ionic
-rw-------    1 obuckley  staff      35 Apr  3  2015 .lesshst
drwxr-xr-x    3 obuckley  staff     102 Dec 25  2014 .local
drwxr-xr-x    3 obuckley  staff     102 Jan 21  2015 .m2
-rw-------    1 obuckley  staff     206 Mar  4  2015 .mysql_history
drwxr-xr-x    4 obuckley  staff     136 Mar 11  2015 .nds
drwxr-xr-x    3 obuckley  staff     102 Jan  5  2015 .node-gyp
drwxr-xr-x  809 obuckley  staff   27506 Nov  3 11:49 .npm
-rw-r--r--    1 obuckley  staff     100 Apr  4  2015 .npmrc
drwxr-xr-x    3 obuckley  staff     102 Aug 26 22:08 .oracle_jre_usage
drwxr-xr-x    4 obuckley  staff     136 Sep 28 13:44 .plugman
drwx------    6 obuckley  staff     204 Oct 20 12:56 .ssh
drwxr-xr-x    6 obuckley  staff     204 Jan 15  2015 .subversion
drwx------   12 obuckley  staff     408 May 15 17:17 .tblive-4
-rw-r--r--    1 obuckley  staff    4152 Apr  4  2015 .v8flags.3.14.5.9.obuckley.json
drwxr-xr-x    9 obuckley  staff     306 Sep 20 20:10 .vagrant.d
drwxr-xr-x    3 obuckley  staff     102 Jul 14 16:38 .vim
-rw-------    1 obuckley  staff   18098 Oct 27 22:27 .viminfo
drwx------    5 obuckley  staff     170 Dec 25  2014 Applications
drwxrwxrwx    7 obuckley  staff     238 Jan  5  2015 Backups
drwxr-xr-x    3 obuckley  staff     102 Jan  5  2015 Books
drwx------@   3 obuckley  staff     102 Dec 25  2014 Creative Cloud Files
drwx------+  11 obuckley  staff     374 Nov  2 20:45 Desktop
drwx------+   9 obuckley  staff     306 Oct 15 08:15 Documents
drwx------+   9 obuckley  staff     306 Oct 28 14:58 Downloads
drwx------@   9 obuckley  staff     306 Nov  3 08:34 Dropbox
drwx------+  61 obuckley  staff    2074 Oct 30 09:13 Library
drwx------+   4 obuckley  staff     136 May 26 09:42 Movies
drwx------+   5 obuckley  staff     170 Jan  5  2015 Music
drwxr--r--    3 obuckley  staff     102 Aug 14 11:01 MyJabberFiles
drwx------+  17 obuckley  staff     578 Sep 23 17:23 Pictures
drwxr-xr-x+   5 obuckley  staff     170 Dec 25  2014 Public
drwx------   11 obuckley  staff     374 Aug 22 17:37 VirtualBox VMs
drwxr-xr-x   48 obuckley  staff    1632 Oct 15 08:12 Workspace
-rw-r--r--    1 obuckley  staff     776 Oct 13 11:19 npm-debug.log
```

 and no file like that in the project I was trying to install it in
```
$ ls -la
total 72
drwxr-xr-x  18 obuckley  staff   612 Nov  2 08:42 .
drwxr-xr-x   3 obuckley  staff   102 Oct 20 12:57 ..
-rw-r--r--   1 obuckley  staff    30 Oct 30 12:14 .csslintrc
-rw-r--r--   1 obuckley  staff   173 Oct 22 18:17 .editorconfig
drwxr-xr-x  14 obuckley  staff   476 Nov  2 08:44 .git
-rw-r--r--   1 obuckley  staff   120 Oct 30 12:33 .gitattributes
-rw-r--r--   1 obuckley  staff    58 Oct 30 11:25 .gitignore
drwxr-xr-x  11 obuckley  staff   374 Nov  2 08:23 .idea
-rw-r--r--   1 obuckley  staff   967 Oct 30 12:48 README.md
drwxr-xr-x   3 obuckley  staff   102 Oct 22 18:14 bin
-rw-r--r--   1 obuckley  staff   496 Oct 30 12:30 bower.json
drwxr-xr-x   7 obuckley  staff   238 Oct 22 18:16 bower_components
drwxr-xr-x   6 obuckley  staff   204 Nov  2 08:42 dest
-rw-r--r--   1 obuckley  staff  2047 Nov  2 08:40 gulpfile.js
drwxr-xr-x  10 obuckley  staff   340 Oct 30 12:32 node_modules
-rw-r--r--   1 obuckley  staff   693 Oct 30 12:33 package.json
-rw-r--r--   1 obuckley  staff    25 Oct 30 10:42 robots.txt
drwxr-xr-x   4 obuckley  staff   136 Oct 30 10:57 src
```

**Comment by thisgeek** [comment]
I was looking for the contents of:
```
-rw-r--r--    1 obuckley  staff     100 Apr  4  2015 .npmrc
```

**Comment by obuckley-kenzan** [comment]
This now works, and will be resolved as part of #6 
`npm install git://github.com/kenzanlabs/pipeline-validate-css.git --save-dev`

Ultimately, we need to get these packages into NPM, however
